lib/utils.ts

```diff
@@ -122,7 +122,7 @@ export function visualize(value: any, depth = 0): string {
 		return `${constructor} (${value.size}) { ${mapped.join(', ')} }`;
 	case 'Set':
 		if (depth > 2) return `Set`;
-		return `${constructor} (${value.size}) { ${[...value].map(v => visualize(v), depth + 1).join(', ')} }`;
+		return `${constructor} (${value.size}) { ${[...value].map(v => visualize(v, depth + 1)).join(', ')} }`;
```

Comment:
Nice catch

Reply:
It's not a comma operator, the argument that should be being passed to `visualize` is being passed to `map` instead (which is completely type legal too, so the compiler won't complain either).

Reply:
> It's not a comma operator, the argument that should be being passed to `visualize` is being passed to `map` instead (which is completely type legal too, so the compiler won't complain either).

> [thisArg Optional](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map#thisarg)
    A value to use as this when executing callbackFn. See [iterative methods](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array#iterative_methods).

I totally forgot about this.